app/.storybook/preview.js

```diff
@@ -10,7 +10,7 @@ export const parameters = {
       order: [
         'Intro',
         'Docs', ['Install', 'Deploy', 'Authentication', 'Fetching', 'Media', 'Tests'],
-        'Components'
+        'Components', ['Avatar', 'Breadcrum', 'Button', 'ConfirmationPrompt', 'Features', 'Forms', 'GapAnalysis', 'Icon', 'InfoButton','Loading', 'Map', 'Modal', 'ProgressBar', 'Projects', 'Scenarios', 'Search', 'Steps', 'Table', 'Tabs', 'Tag', 'Toast', 'Tooltip'], 
```

Comment:
Why are you adding this order if at the end you are sorting them by name? Could you remove it?